cfgov/v1/atomic_elements/organisms.py

```diff
@@ -5,6 +5,7 @@
 from . import atoms, molecules
 from ..util import ref
 from ..models.snippets import Contact as ContactSnippetClass
+from wagtail.contrib.table_block.blocks import TableBlock
```

Comment:
Nitpicky, but can third-party imports go in a separate group before our own imports? See the second bullet in PEP8 [here](https://www.python.org/dev/peps/pep-0008/#imports).